packages/EasyDoctrine/docs/doctrine/entity_manager_decorator.md

```diff
@@ -112,6 +112,8 @@ final class SomeService
     public function someMethod(): void
     {
         $this->deferredEntityEventDispatcher->disable();
+        // Disabling only for passed entity classes:
```

Comment:
Pls rewrite it to:
`to disable only certain entities, use the following call:`

Reply:
Done